build-system/tasks/runtime-test/helpers.js

```diff
@@ -340,7 +341,7 @@ async function runTestInBatches_(config, browsers) {
       'beta',
       browsers.beta,
       config,
-      /* runCompleteFn */ () => {}
+      reportTestRunComplete
```

Comment:
With the new `beta` test status, I'm not sure this works for batch testing, because `reportTestStarted()` is called once with `integration/saucelabs` as the check name, but `reportTestRunComplete()` is called twice with `integration/saucelabs-stable` and `integration/saucelabs-beta` check names.

Reply:
Fair point. I've tried fixing this with minimal changes in 9ae7164ab8a07c21062032c2149047a1d7236960. (f stable fails, report immediately and exit. Otherwise, continue count while running beta and then report.)